src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditFormPlaceholder.tsx

```diff
@@ -1,20 +1,49 @@
 import {
-  Box,
   Flex,
   Text,
   Spacer,
   SkeletonText,
   SkeletonBox,
+  Skeleton,
 } from "@artsy/palette"
+import { AlertProvider } from "Components/Alert/AlertProvider"
 import { CriteriaPillsPlaceholder } from "Components/Alert/Components/CriteriaPills"
+import { Modal } from "Components/Alert/Components/Modal/Modal"
 import { Media } from "Utils/Responsive"
 
-export const SavedSearchAlertEditFormPlaceholder = () => {
+export const SavedSearchAlertEditFormPlaceholder: React.FC<{
+  onCloseClick?: () => void
+}> = ({ onCloseClick }) => {
   return (
-    <Box flex={1} p={4}>
-      <Text variant="lg" mb={4}>
-        Edit Alert
-      </Text>
+    <>
+      <Media greaterThanOrEqual="md">
+        <Skeleton flex={1} p={4}>
+          <SavedSearchAlertEditFormPlaceholderContext />
+        </Skeleton>
+      </Media>
+      <Media lessThan="md">
+        <AlertProvider isEditMode>
```

Comment:
**question**: is there a need for AlertProvider in the placeholder component?

Reply:
I chose to use it to render the correct header
When working on alerts in the last iteration we added header component for mobile and I decided to reuse it to stay consistent with the UI and avoid copy and paste of code
I'm pretty sure the whole implementation can se simplified, but it's a task for another day :)